package/client/resource/dui/index.ts

```diff
@@ -0,0 +1,80 @@
+import {
+	GetGameTimer,
+	CreateDui,
+	GetDuiHandle,
+	CreateRuntimeTxd,
+	CreateRuntimeTextureFromDuiHandle,
+	SetDuiUrl,
+	DestroyDui,
+	SendDuiMessage,
+} from '@nativewrappers/client';
```

Comment:
These types are globally defined from the `citizenfx/client` package and shouldn't be imported (and don't exist at the given path).